packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,68 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
 ---
 
 ## Examples
+
+### Variant examples
+
+Alert groups utilize the same variants that are available to [alert components](components/alert/). You can add multiple alert variants to each group.
+
 ### Static alert group
-These alerts appear on page load and are discoverable from within the normal page content flow, and will not be announced individually/explicitly to assistive technology.
+
+Static alert groups appear when the page loads. They are seen within the normal page content flow. They will not be announced individually or explicitly to assistive technology.
```

Comment:
Since all alert groups themselves should appear on page load, maybe something along the lines of:

```suggestion
Static alert groups contain alerts that appear when the page loads, and are seen within the normal page content flow. Static alert groups should not contain alerts that will dynamically appear or update.
```